.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
+es5.1/
```

Comment:
I used both pdfs on my local system(for extra/better testing). We can use it(`helloworld.pdf`) in other tests.